doc/changelog.rst

```diff
@@ -188,6 +188,7 @@ Released on 2016-01-21 11:15 +0200.
 Core
 ~~~~
 
+- Add ``get_company_contact`` to ``shoop.core.models``
```

Comment:
This shouldn't be under 3.0.0 release. Also you're missing the entry for Front ("Log-in as company when user is member of a company").